tests/src/test/java/alluxio/server/ft/journal/raft/EmbeddedJournalIntegrationTest.java

```diff
@@ -172,7 +172,6 @@ public void restart() throws Exception {
     assertTrue(fs.exists(testDir));
     restartMasters();
     assertTrue(fs.exists(testDir));
-    mCluster.saveWorkdir();
```

Comment:
Was this previously a bug/left for debugging purposes?

Reply:
Most probably left from a debugging session. It exists in the original commit for adding the test.